content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -110,86 +51,57 @@ follow
-read more about the flags `kubeadm init` provides. You can also specify a
-[configuration file](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file) instead of using flags.
-- You need to choose a Pod Network Plugin in the next step. Depending on what
-third-party provider you choose, you might have to set the `--pod-network-cidr` to
-something provider-specific. The tabs below will contain a notice about what flags
-on `kubeadm init` are required.
-- Unless otherwise specified, kubeadm uses the default gateway's network interface
-to advertise the master's IP. If you want to use a different network interface, specify
-`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you must specify an IPv6, e.g. `--apiserver-advertise-address=fd00::101`
-- If you would like to customise control plane components including optional IPv6 assignment to liveness probe for control plane components and etcd server, you can do so by providing extra args to each one, as documented [here](/docs/admin/kubeadm#custom-args).
-- `kubeadm init` will first run a series of prechecks to ensure that the machine
-is ready to run Kubernetes. It will expose warnings and exit on errors. It
-will then download and install the cluster database and control plane
-components. This may take several minutes.
-- You can't run `kubeadm init` twice without tearing down the cluster in between
-([unless you're upgrading from v1.6 to v1.7](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)),
-see [Tear Down](#tear-down).
+- For more information about `kubeadm init` arguments, see [the kubeadm reference guide.](/docs/reference/setup-tools/kubeadm/kubeadm/) 
+- For a complete list of configuration options, see [the configuration file documentation.](/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file)
+- To customise control plane components, including optional IPv6 assignment to liveness probe for control plane components and etcd server, you provide extra arguments to each component. See  as documented [here](/docs/admin/kubeadm#custom-args).
+- To run `kubeadm init` again, you must first [tear down the cluster](#tear-down).
 - If you join a node with a different architecture to your cluster, you should create a separate
 Deployment or DaemonSet for `kube-proxy` and `kube-dns` on the node. This is because the Docker images for these
 components do not currently support multi-architecture.
 
+`kubeadm init` first runs a series of prechecks to ensure that the machine
+is ready to run Kubernetes. It exposes warnings and exits on errors. It
+then downloads and installs the cluster control plane components. 
+This may take several minutes.
+
 The output should look like:
 
 ```
-[init] Using Kubernetes version: v1.8.0
-[init] Using Authorization modes: [Node RBAC]
+[init] Using Kubernetes version: vX.Y.Z
```

Comment:
I think we have a variable for this. @Bradamant3 can you point to any info about this variable? I'm not sure where that is documented.